### PR TITLE
fix: manual review of #3357

### PR DESCRIPTION
**Pull request by pdobrien3**

### CODE REVIEW

**Comment by pdobrien3** [comment]
Ha, obviously I failed miserably?

**Comment by AlCalzone** [comment]
Not sure if you see the check results, but there is at least one syntax error:
```
Error:  packages/config/config/devices/0x014f/wo15emz5.json: SyntaxError: Unexpected token, expected "," (69:5)
Error:    67 | 					"value": 0
Error:    68 | 				}
Error:  > 69 | 				{
Error:       | 				^
Error:    70 | 					"label": "minutes",
Error:    71 | 					"value": 1.255
Error:    72 | 				}
```

If you're working locally, you can execute `yarn run lint` to check before pushing.

**Comment by pdobrien3** [comment]
I am using vscode and have no idea what the syntax error is.  Oh, it needs a comma somewhere.  I fixed it.  Obviously I dont create a new pull request.  How do I now merge my changes?

Auh, it pushed up stream after I made the change ;-)  And started the checks again.

**Comment by AlCalzone** [comment]
Now the formatting is off. Not a huge deal, we can fix that before merging right here on Github.

File: packages/config/config/devices/0x014f/wo15emz5.json [hunk 1]
```diff
@@ -33,47 +32,51 @@
...
 			"options": [
 				{
-					"label": "Automatic reporting off",
+					"label": "disable",
 					"value": 0
+				},
+				{
+					"label": "minutes",
+					"value": 1.255
 				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x014f/wo15emz5.json]: Remove this option, add `"unit": "minutes"` to the parameter definition.

File: packages/config/config/devices/0x014f/wo15emz5.json [hunk 2]
```diff
@@ -33,47 +32,51 @@
...
+			"description": "Automatic Reporting Interval",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 255,
 			"defaultValue": 6,
 			"unsigned": true,
 			"options": [
 				{
-					"label": "Automatic reporting off",
+					"label": "disable",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x014f/wo15emz5.json]:
```suggestion
					"label": "Disable",
```

**Comment by pdobrien3** [comment]
I appreciate that.  If you wouldn't mind, could you at least tell me what was wrong so I learn from it?  Also, does that mean everything is done on my end?

File: packages/config/config/devices/0x014f/wo15emz5.json [hunk 3]
```diff
@@ -33,47 +32,51 @@
...
 					"value": 2
 				},
 				{
-					"label": "Led is always off",
+					"label": "Always off",
 					"value": 3
 				},
 				{
-					"label": "Led blinks during rf transmissions from or to the device",
+					"label": "Blinks during rf transmissions",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x014f/wo15emz5.json]:
```suggestion
					"label": "Blink during RF transmissions",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
A couple of changes, but looks pretty good so far.

>  If you wouldn't mind, could you at least tell me what was wrong so I learn from it?

I think it is an extra space at the end of a line. If you've installed all dependencies and the recommended extensions, just saving the file in VSCode should take care of it.

File: packages/config/config/devices/0x014f/wo15emz5.json [hunk 4]
```diff
@@ -14,7 +14,7 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
-	"associations": {
+	"associations": {
```
Comment (AlCalzone) [inline packages/config/config/devices/0x014f/wo15emz5.json]: FYI, here is the extra space. I'll have the bot fix it.

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
Thanks

**Pull request merged**